thesis-api-v2/pom.xml

```diff
@@ -91,6 +101,11 @@
 			<artifactId>bcprov-jdk15on</artifactId>
 			<version>1.70</version>
 		</dependency>
+	    	<dependency>
+	      		<groupId>com.vaadin.external.google</groupId>
+	      		<artifactId>android-json</artifactId>
+	      		<version>0.0.20131108.vaadin1</version>
+	    	</dependency>
```

Comment:
Please remove this dependency. Spring already comes with [Jackson](https://github.com/FasterXML/jackson-core) so there is no need to add another JSON parsing lib.

Reply:
To be honest I just picked the first dependency InteliJ suggested for parsing Json's. I will look at Jackson as you suggested.

Reply:
Removed `android-json` dependency, replaced with Jackson as suggested.